cmds/ocm/commands/ocmcmds/components/hash/cmd.go

```diff
@@ -102,8 +102,8 @@ func TableOutput(opts *output.Options, h *action, mapping processing.MappingFunc
 	def := &output.TableOutput{
 		Headers: output.Fields("COMPONENT", "VERSION", "HASH", wide),
 		Options: opts,
-		Chain:   comphdlr.Sort.Map(h.digester),
-		Mapping: mapping,
+		Chain:   comphdlr.Sort,
+		Mapping: processing.MappingSequence(h.digester, mapping),
```

Comment:
The current logic needs rework urgently, but the since the fix is simple, it's alright for now.